geode-redis/src/main/java/org/apache/geode/redis/internal/KeyRegistrar.java

```diff
@@ -76,16 +76,26 @@ public void validate(ByteArrayWrapper key, RedisDataType type) {
     }
   }
 
+  /**
+   * Checks if the given key is a protected string in GeodeRedis
+   *
+   * @param key Key to check
+   */
+  public boolean isProtected(ByteArrayWrapper key) {
+    return redisMetaRegion.containsKey(key.toString())
```

Comment:
I think the extra containsKey call is not needed in this get. Region.get will return null if a region does not contain the key and null will not be equal to REDIS_PROTECTED.
So I advise you get rid of the containsKey call.

Reply:
Good call Darrel. Addressed that.